Clinical-Genomics/preClinVar
# Parse variant coordinates, copy number and variant type

### This PR adds | fixes:
- Fix #67

### Added
- Parsing of `variantType`, `referenceCopyNumber`, `copyNumber` and `chromosomeCoordinates` fields from CSV/TSV files
- Genome assembly can be passed as an optional param to tsv_2_json and csv_to_json to create submission containing chromosome coordinates

### How to test:
Submit a Variant file with the following variants 
- A SNV without accession or HGVS (coordinates should be parsed correctly -> should have a start and a stop)
-  A SV with Breakpoint1 and Breakpoint2 (coordinates should be parsed correctly -> should have a start and a stop)
-  A SV with uncertain coordinates (outer start, inner start etc)
- A SV with reference copy number and alt copy number
- A SV of a certain type (Del, Ins etc)

### Expected outcome:
- All variants should be parsed correctly, given that the request is sent providing a genome assembly

### Review:

This [version](https://semver.org/) is a:
